inst/extdata/genParBaselinePrefTrends.csv

```diff
@@ -6343,19 +6343,19 @@ SSP1;FRA;Liquids;Truck (26t);trn_freight_road_tmp_subsectorL3;trn_freight_road_t
 SSP1;FRA;Liquids;Truck (40t);trn_freight_road_tmp_subsectorL3;trn_freight_road_tmp_subsectorL2;trn_freight_road;trn_freight;FV;1;1;1;0.5672;0.5672
 SSP1;FRA;Liquids;Truck (7_5t);trn_freight_road_tmp_subsectorL3;trn_freight_road_tmp_subsectorL2;trn_freight_road;trn_freight;FV;1;1;0.7518;0.5502;0.5502
 SSP1;IND;;;;;Cycle;trn_pass;S1S;0.01915;0.04615;0.075;0.2;0.2
-SSP1;IND;;;;;Domestic Aviation;trn_pass;S1S;0.6659;0.2753;0.1312;0.04772;0.04772
+SSP1;IND;;;;;Domestic Aviation;trn_pass;S1S;1.4;0.2753;0.1312;0.04772;0.04772
 SSP1;IND;;;;;Domestic Ship;trn_freight;S1S;0.001202;0.001202;0.001092;0.0009244;0.0009244
 SSP1;IND;;;;;Freight Rail;trn_freight;S1S;0.1924;0.2886;0.2361;0.1998;0.1998
-SSP1;IND;;;;;HSR;trn_pass;S1S;0;0.0004615;0.00055;0.0003;0.0003
+SSP1;IND;;;;;HSR;trn_pass;S1S;0;0.0004615;0.00055;3E-04;3E-04
```

Comment:
please remove all "unwanted" changes next time before comitting

Reply:
Like this its very hard to make sure there are no actual changes comitted, that are unwanted